### DISCUSSION TITLE
効果音の要望

### DISCUSSION THREAD

**Discussion by multilockon123** [hengband/hengband#1183] [Ideas]
お疲れ様です。
装備品を装備した時の効果音を所望します。
装備の更新はレベルアップに並ぶ節目なので…

**Comment by shimitei** [comment]
装備した/装備を外した時の効果音を追加しました。
#1213 #1214 
次のAlpha28リリースに含まれる予定です。

**Comment by multilockon123** [comment]
お疲れ様です。
階段の昇降、広域マップと通常マップの行き来、マップ端からの脱出、クエスト侵入など
マップが切り替わった際（レベルテレポ除く）の効果音をお願いします。

**Comment by shimitei** [comment]
階段の昇降とクエスト侵入はコマンドも同じなので同じ効果音で問題なさそう。

広域マップと通常マップの行き来は、どういう効果音がふさわしいか思い浮かばない。ゲームによくある機能でないから？
マップ端からの脱出も本来は地続きなのだから特別な効果音は無しかな。

**Comment by ghost** [comment]
### 呪われた時の効果音2種追加要望
「軽い呪い(解呪で解ける)」と「重い呪い(*解呪*で解ける)」が@の装備に掛かった時それぞれの効果音が欲しいです。
両方掛かっている時に「解呪」の「誰かに見守られてる～」メッセージで全部解けたつもりになって、実は重い方(反感)が残ったままなのに気づかず鉄獄深層に帰還して死にかけた事があったので。
呪いをかけられたタイミングで軽重どちらか判断出来たら便利かなと思いました。
ご検討いただければ幸いです。

**Comment by ghost** [comment]
### 警告の効果音追加要望
ユニークの生成を検知する「～色に光った」のタイミングで効果音が欲しいです。
(ユニーク生成は移動中に不意に出てきたりで割と見落とすことが多いので。）
なおその他のトラップやモンスターの隣(?)へ移動する直前などにも警告が発せられますが、
y/nの問い合わせが出るため見落としはほぼ無いと思われ、これらは音を出すニーズは低いかと個人的には思っています。
ご検討いただければ幸いです。

**Comment by chikatei** [comment]
打撃や射撃と同じように、攻撃魔法が命中した際にも効果音が鳴ると嬉しいです。
もっとも魔法は種類が多いので効果音を何種類に分けるかが問題ですが……。火炎などの属性別に分ける、ボールとボルト状などの属性別に分ける、もしくは単に魔法攻撃というくくりで一種類だけ用意する、などでもよさそうです。

**Comment by multilockon123** [comment]
お疲れ様です。
既に出ている魔法攻撃の案についても掘り下げてみました。

・歩行音（走るときは何歩でも１度鳴る）
・ペットの攻撃がヒットする音
・魔法攻撃の音
　・ボルト系（反射で跳ね返せるもの）
　・ボール系（ボール、嵐、ロケット、放射能球、純ログルス、悪臭雲、地獄球、スターバースト、メテオスウォームなど炸裂するもの）
　・光の剣
　・その他（軽傷、重症、致命傷、脳攻撃、秘孔、破滅の手など）
・時を止める音
・無敵化する音
・*破壊*の音
・エキサイトモンスター、悲鳴の音
・毒針による即死、忍者ヒットの音

**Comment by nan9090** [comment]
自動破壊はこのゲームの特徴だと思うので効果音欲しい
アイテムいっぱい落ちてガシガシ壊すのは効果音あると楽しくなりそう

＊破壊＊もこのゲームの象徴なので効果音欲しい

狂戦士化もテンション上がるので欲しい

文字列に反応して効果音鳴らすようにするのは難しいですか？
例えば、「破壊」という文字が表示されたらこの音を鳴らす、というのをプレーヤーが自分で決めることができるように
（確かダンジョンクロールがそんな感じだったような）
各々勝手に改造しろ的な感じで

**Comment by avutkl** [comment]
ザックから物があふれたときの効果音の実装をお願いします。
これは特に杖でやらかしがちなんで。
